src/dashboard/trulens/dashboard/run.py

```diff
@@ -167,18 +166,18 @@ def listen_to_tunnel(proc: subprocess.Popen, pipe, out, started):
                 else:
                     print(line)
 
-        Tru.tunnel_listener_stdout = Thread(
+        session.tunnel_listener_stdout = Thread(
             target=listen_to_tunnel,
             args=(tunnel_proc, tunnel_proc.stdout, out_stdout, tunnel_started),
         )
-        Tru.tunnel_listener_stderr = Thread(
+        session.tunnel_listener_stderr = Thread(
             target=listen_to_tunnel,
             args=(tunnel_proc, tunnel_proc.stderr, out_stderr, tunnel_started),
         )
-        Tru.tunnel_listener_stdout.daemon = True
-        Tru.tunnel_listener_stderr.daemon = True
-        Tru.tunnel_listener_stdout.start()
-        Tru.tunnel_listener_stderr.start()
+        session.tunnel_listener_stdout.daemon = True
+        session.tunnel_listener_stderr.daemon = True
+        session.tunnel_listener_stdout.start()
+        session.tunnel_listener_stderr.start()
         if not tunnel_started.wait(
```

Comment:
Same with many of the places here. This was earlier referring to class members and now object members.